front/src/applications/operationalStudies/hooks/useScenarioData.ts

```diff
@@ -178,14 +171,21 @@ const useScenarioData = (
     selectedTrainId,
     trainScheduleSummaries,
     trainSchedules,
-    trainScheduleUsedForProjection,
-    trainIdUsedForProjection,
-    projectedTrains,
-    allTrainsProjected,
+    projection:
```

Comment:
Not really important if you don't want to :)

```suggestion
    projectionData:
```

Reply:
Done